src/components/ButtonWithDropdownMenu.js

```diff
@@ -54,16 +54,26 @@ const ButtonWithDropdownMenu = (props) => {
     const [popoverAnchorPosition, setPopoverAnchorPosition] = useState(null);
     const {windowWidth, windowHeight} = useWindowDimensions();
     const caretButton = useRef(null);
-    useEffect(() => {
-        if (!caretButton.current) {
-            return;
-        }
+
+    const measurePopoverPosition = () => {
         caretButton.current.measureInWindow((x, y, w, h) => {
             setPopoverAnchorPosition({
                 horizontal: x + w,
                 vertical: y + h,
             });
+            setIsMenuVisible(true);
         });
+    };
+
+    useEffect(() => {
+        if (!caretButton.current) {
+            return;
+        }
+        if (popoverAnchorPosition !== null) {
+            measurePopoverPosition();
+        }
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@s77rt  because it asks to add popoverAnchorPosition as a dependency which I think we should not add here.
<img width="928" alt="Screenshot 2023-06-13 at 23 49 11" src="https://github.com/Expensify/App/assets/58285468/48a717c7-295c-49bc-9154-b56463b99524">

Reply:
@s77rt  actually you requested to remove popoverAnchorPosition null check from here I removed it so we do not need es-lint disabling rule anymore